setup.py

```diff
@@ -44,10 +44,10 @@ def find_version(*file_paths):
     license='MIT',
     packages=[
         'buildozer', 'buildozer.targets', 'buildozer.libs', 'buildozer.scripts'
-    ],
+        ],
```

Comment:
Could you please remove this empty change?

Reply:
Yes I also got bugged by it, but I tried to not say anything because I'm always the one nit picking :stuck_out_tongue: 
So sorry if I saw your comment before I would not have merged :confused: